Tasks/Src/RTOS_BPS.c

```diff
@@ -4,21 +4,138 @@
 
 #include "RTOS_BPS.h"
 
-void RTOS_BPS_SemPend(void){ 
+/**
```

Comment:
All the BPS RTOS functions should have the Fault Flag variable used (skip the function if the Fault Flag variable is set).